node-notes-cli/app.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-console */
+
+// write to data.json
+// read function
+
+const fs = require('fs');
+const dataJson = require('./data.json');
+if (process.argv[2] === 'read') {
+
+  fs.readFile('./data.json', 'utf8', (err, data) => {
```

Comment:
since your `require()`d the `data.json`, you can skip reading the file and just loop through the notes.